### DISCUSSION TITLE
Non-coding genes

### DISCUSSION THREAD

**Discussion by damhanalla** [churchmanlab/genewalk#72] [Q&A] [answered]
Hello and thank you for developing GeneWalk. I wanted to ask if this program works with non-coding genes, as I have read that if a gene does not have an uniprot ID associated, it won't be included in the network.

Thank you very much for your time

**Comment by ri23** [comment, marked as answer]
Hi @ansionnachfionn ,
Thank you for reaching out. Indeed, GeneWalk currently does not support non-coding genes for the reasons you mentioned. It is possible to extend GeneWalk to non-coding genes. All that is needed are GO annotations of the non-coding RNA and a knowledge base that provides interactions of the RNA with proteins and/or other RNAs. But developing this is not in our plans for the foreseeable future. If you are interested in developing this functionality, feel free to do so of course. You can contact me via email in case you would like some advice or potentially to collaborate.
